<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AboutDialogClass</class>
 <widget class="QDialog" name="AboutDialogClass">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>577</width>
    <height>599</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>About</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tabAbout">
      <attribute name="icon">
       <iconset resource="../res/diffimg.qrc">
        <normaloff>:/diffimg/about.png</normaloff>:/diffimg/about.png</iconset>
      </attribute>
      <attribute name="title">
       <string>About</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_7">
       <item row="1" column="0">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="label">
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="../res/diffimg.qrc">:/diffimg/splashscreen.png</pixmap>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="0" colspan="3">
        <widget class="QTabWidget" name="tabWidget_2">
         <property name="tabPosition">
          <enum>QTabWidget::South</enum>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab">
          <attribute name="title">
           <string>Credits</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_6">
           <item row="0" column="0">
            <widget class="QTextEdit" name="textEditCredits">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="font">
              <font>
               <pointsize>8</pointsize>
              </font>
             </property>
             <property name="autoFillBackground">
              <bool>false</bool>
             </property>
             <property name="frameShape">
              <enum>QFrame::NoFrame</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Plain</enum>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_2">
          <attribute name="title">
           <string>Changelog</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QTextEdit" name="textEditChangelog">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="font">
              <font>
               <pointsize>8</pointsize>
              </font>
             </property>
             <property name="autoFillBackground">
              <bool>false</bool>
             </property>
             <property name="frameShape">
              <enum>QFrame::NoFrame</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Plain</enum>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item row="0" column="0" colspan="3">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="labelRelease">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Release Number</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0" colspan="3">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabPreferences">
      <attribute name="icon">
       <iconset resource="../res/diffimg.qrc">
        <normaloff>:/diffimg/preferences.png</normaloff>:/diffimg/preferences.png</iconset>
      </attribute>
      <attribute name="title">
       <string>Preferences</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="7" column="0" colspan="5">
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="5">
        <widget class="QGroupBox" name="groupBoxLang">
         <property name="title">
          <string>Language</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_14">
          <item row="0" column="0">
           <widget class="QComboBox" name="comboBoxLanguage"/>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="labelDetectedLanguage">
            <property name="text">
             <string>Detected language ...</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="9" column="0" colspan="5">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Difference metric used</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_10">
          <item row="3" column="0" colspan="3">
           <widget class="QGroupBox" name="groupBoxEnableThreshold">
            <property name="title">
             <string>Allow image difference</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <layout class="QGridLayout" name="gridLayout_11">
             <item row="0" column="0">
              <widget class="QLabel" name="label_5">
               <property name="toolTip">
                <string>Discriminating parameter (Image are considered as different if the threshold is exceeded)</string>
               </property>
               <property name="text">
                <string>Discriminating parameter</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>Threshold</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="lineEditThreshold"/>
             </item>
             <item row="1" column="2">
              <widget class="QPushButton" name="pushButtonApplyImageParam">
               <property name="toolTip">
                <string>Apply</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="../res/diffimg.qrc">
                 <normaloff>:/diffimg/check2.png</normaloff>:/diffimg/check2.png</iconset>
               </property>
              </widget>
             </item>
             <item row="0" column="1" colspan="2">
              <widget class="QComboBox" name="comboBoxOutputParam"/>
             </item>
            </layout>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Metric</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QLabel" name="labelMetricDesc">
            <property name="text">
             <string/>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="2" rowspan="2">
           <widget class="QLabel" name="labelMetricPixmap">
            <property name="maximumSize">
             <size>
              <width>48</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="comboBoxMetrics"/>
          </item>
          <item row="2" column="0" colspan="3">
           <widget class="QGroupBox" name="groupBox_3">
            <property name="title">
             <string>Input parameters</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_12" columnstretch="1,0,0">
             <item row="0" column="0">
              <widget class="QComboBox" name="comboBoxInputParam"/>
             </item>
             <item row="0" column="2">
              <widget class="QPushButton" name="pushButtonApplyInputParam">
               <property name="toolTip">
                <string>Apply</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="../res/diffimg.qrc">
                 <normaloff>:/diffimg/check2.png</normaloff>:/diffimg/check2.png</iconset>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="lineEditInputParam"/>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="6" column="2">
        <widget class="QSlider" name="horizontalSliderOpacity">
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="sliderPosition">
          <number>50</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="6" column="3" colspan="2">
        <widget class="QSpinBox" name="spinBoxOpacity">
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="value">
          <number>50</number>
         </property>
        </widget>
       </item>
       <item row="11" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>5</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="6" column="0" colspan="2">
        <widget class="QLabel" name="label_13">
         <property name="toolTip">
          <string>Mask opacity (0-transparent,100-opaque)</string>
         </property>
         <property name="text">
          <string>Error mask opacity (%)</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Wipe effect axis</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2" colspan="3">
        <widget class="QComboBox" name="comboBoxWipeEffectAxis"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabInfos">
      <attribute name="icon">
       <iconset resource="../res/diffimg.qrc">
        <normaloff>:/diffimg/help.png</normaloff>:/diffimg/help.png</iconset>
      </attribute>
      <attribute name="title">
       <string>Infos</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0">
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="title">
            <string>Supported formats (via Qt plugins)</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_8">
            <item row="0" column="0">
             <widget class="QListWidget" name="listWidgetFormats">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="alternatingRowColors">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="labelQtVersion">
              <property name="font">
               <font>
                <pointsize>7</pointsize>
               </font>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_4">
           <property name="title">
            <string>Available translations</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_9">
            <item row="0" column="0">
             <widget class="QListWidget" name="listWidgetTranslations">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="alternatingRowColors">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="labelTranslation">
              <property name="font">
               <font>
                <pointsize>7</pointsize>
               </font>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabLog">
      <attribute name="icon">
       <iconset resource="../res/diffimg.qrc">
        <normaloff>:/diffimg/journal.png</normaloff>:/diffimg/journal.png</iconset>
      </attribute>
      <attribute name="title">
       <string>Log</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="1" column="0" colspan="2">
        <widget class="QTextBrowser" name="textBrowserLog"/>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="pushButtonClearLog">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../res/diffimg.qrc">
           <normaloff>:/diffimg/delete.png</normaloff>:/diffimg/delete.png</iconset>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../res/diffimg.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AboutDialogClass</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>266</x>
     <y>480</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AboutDialogClass</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>334</x>
     <y>480</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderOpacity</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBoxOpacity</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>474</x>
     <y>166</y>
    </hint>
    <hint type="destinationlabel">
     <x>522</x>
     <y>167</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxOpacity</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSliderOpacity</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>522</x>
     <y>167</y>
    </hint>
    <hint type="destinationlabel">
     <x>474</x>
     <y>166</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
